src/screens/WebScreen.js

```diff
@@ -24,6 +26,21 @@ export const WebScreen = ({ route }) => {
     isConnected && webUrl && trackScreenViewAsync(`${MATOMO_TRACKING.SCREEN_VIEW.WEB} / ${webUrl}`);
   }, [webUrl]);
 
+  useEffect(() => {
+    if (documentData?.title) {
+      navigation.setOptions({
+        headerRight: () => (
+          <TouchableOpacity
+            style={styles.headerRight}
+            onPress={() => onDownloadAndSharePdf(documentData)}
+          >
+            <Icon.ArrowDownCircle color={colors.lightestText} />
+          </TouchableOpacity>
```

Comment:
the padding looks a bit too tight. i will adjust this according to all other header right elements:

<img width="381" alt="image" src="https://github.com/user-attachments/assets/cf42f05a-4c75-4114-9120-a2e8453076b4">